themes/bootstrap3/scss/components/search.scss

```diff
@@ -88,9 +88,19 @@ header .container.navbar { margin-bottom: 0; }
   margin: 0;
   padding: 0;
   list-style: none;
+
+  .popover ul {
+    list-style: disc;
+    padding-left: 1em;
+  }
 }
 .action-toolbar li {
   display: inline-block;
+
+  .popover li {
+    display: list-item;
+    padding-left: 0.5em;
+  }
```

Comment:
Do we want to apply this style to ul and li globally?

Reply:
I was afraid it could have unexpected consequences, and in other occasions like larger content areas the normal indentations could be fine. Part of these settings are these just to reverse the unstyled lists inherited from the toolbar, and part makes the indentation suit the small popover area better.

Reply:
Yeah, seems safer to keep this more narrowly scoped, at least for now. In the long term, I think a broader CSS refactor is probably in order. :-)